actions/index.js

```diff
@@ -0,0 +1,7 @@
+export function AddItemAction(modalText, response) {
+  return {
+    type: 'ADD_ITEM',
+    modalText,
```

Comment:
What do you think to specialise everything besides the action type inside a `payload` attribute?
This can be converted into:

```js
return {
  type: 'ADD_REPOSITORY',
  payload: {modalText, ...response}
}
```

This way, at the reducer, you'll access those infos like: `action.payload.modalText`, `action.payload.id` and so on...